### ISSUE TITLE
Unable to instantiate the Kanban

### ISSUE THREAD

**Issue by raj-wadhwa** [Zaloog/kanban-python#13]
Hi,

This is the first time I installed "kanban-python". After installing the package, I ran the following command but got the below error:

> kanban init

```log
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/bin/kanban", line 5, in <module>
    from kanban_python.app import run
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/kanban_python/app.py", line 25, in <module>
    from kanban_python import cli_parser, config, controls, utils
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/kanban_python/controls.py", line 106, in <module>
    def add_tasks_to_db(tasks: dict | list[dict]) -> None:
TypeError: unsupported operand type(s) for |: 'type' and 'types.GenericAlias'
```

What could be the reason for this?

**Comment by Zaloog** [comment]
Hi @raj-wadhwa,

I used a Union type hint in one function and python 3.9 is not able to interpret the Union `|` character correctly with the 2 given types.
This feature was is not availaable in 3.9 *

Upgrading your python version would fix the error. If thats not an options for you I released a new version that adds the
`from __future__ import annotations`  in v0.3.9, this should make it useable also with the older 3.9 version.
